docs/deployment/configuration/files.md

```diff
@@ -264,32 +264,41 @@ artifactRepository:
     secretKey: 5bEYu26084qjSFyclM/f2pz4gviSfoOg+mFwBH39
 ```
 
-GCS and Minio configurations would also be similar except that the endpoint will be different. For GCS, you would need to [create an HMAC key](https://cloud.google.com/storage/docs/authentication/managing-hmackeys#create) and use the **Access key** and **Secret** accordingly.
```

Comment:
MinIO is still an option and should not be removed

Reply:
oh gotcha, will create another PR and cancel this one